src/plugins/stepsform/stepsform.js

```diff
@@ -130,9 +133,59 @@ var componentName = "wb-steps",
 				$( form ).children( "input" ).hide();
 				wb.ready( $( elm ), componentName );
 			}
+
+			//Quiz code
+			if ( elm.classList.contains( "quiz" ) ) {
+
+				//Initialize all instances
+				let $instance = $( elm ),
+					numQuestion = $( ".steps-wrapper", $instance ).length; // Calculate number of questions
+
+				// Addition to UI (Ex: progress bar)
+				if ( !$.contains( $instance, "progress" ) ) {
+					$( "form", $instance ).prepend( "<label class='full-width'><span class='wb-inv'>" + i18nText.progresslabel + "</span><progress class='progressBar' max='" + numQuestion + "'></progress><p class='progressText' role='status'></p></label>" );
+				}
+
+				hideOtherSteps( evt );
+
+				$( document ).on( "click", quizSelector + " .steps-wrapper div.buttons > :button", hideOtherSteps );
+			}
 		}
 	},
 
+	/**
+	 * @method hideOtherSteps
+	 * @param {JavaScript element} e
+	 */
+	hideOtherSteps = function( e ) {
```

Comment:
The name of this function is not really meaningful. I feel like something similar to "updateQuizStep" would be more appropriate.